css_tasks/task11/scss/_clients.scss

```diff
@@ -0,0 +1,99 @@
+@import "variables";
+@import "mixins";
+.clients {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top:6rem;
+  height: 40rem;
+  background: url("../assets/layer-53@3x-min.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+  &:after {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    background: #d9d9d9;
+    opacity: 0.8;
+    content: "";
+  }
+}
+.clients__wrapper {
+  position: absolute;
```

Comment:
why it is here for element which is just in same place where it should be

Reply:
Well, I removed absolute from desktop view, and you are right. But when I switched below 1024px, I changed `display:grid` to `display:block`, and the content was dimmed, below the opacity of the grey background. So I added `z-index` but had to add absolute again. Probably I don't see something. :(